.ci/setup_mysql.sh

```diff
@@ -11,7 +11,7 @@ mysql -h 127.0.0.1 -u root \
 	-e "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));"
 echo "[*] Setting MySQL max_connections..."
 mysql -h 127.0.0.1 -u root \
-	-e "SET GLOBAL max_connections = 200;"
+	-e "SET GLOBAL max_connections = 500;"
```

Comment:
consequence of the way the sender cache is initialized by every api process. Will be refactored in next PR so that cache and subsequently the mysql connection they create is shared across all api processes.